packages/mjml-core/src/index.js

```diff
@@ -104,7 +104,7 @@ export default function mjml2html(mjml, options = {}) {
       break
   }
 
-  const mjBody = traverseMJML(mjml, child => child.tagName === 'mj-body')
+  const mjBody = traverseMJML(mjml, child => child.tagName === 'mj-body', 'mj-attributes')
```

Comment:
Like here

Reply:
When thinking twice as this, I think it would add an hard dep on 'mj-attributes'  in the core. I think you should add a `deepSearch` parameters that would only search in root level childrens